src/js/ia/IAPage.js

```diff
@@ -1522,6 +1527,19 @@
                         });
                     }
 
+                    // Called on userpages links click
+                    function sendReq($obj) {
+                        var username = $.trim($obj.text());
+                        var id = $obj.attr("id").match(/maintainer/)? "maintainer" : "contributor";
+                        var url_suffix = "?" + Math.ceil(Math.random() * 1e7);
+                        var img = new Image();
+
+                        username = username? "_" + username : "";
```

Comment:
Do you want to strip special characters or anything from the username? Or are they safe?